lib/osom_tables.rb

```diff
@@ -1,3 +1,3 @@
 module OsomTables
-  VERSION = '3.1.4'
+  VERSION = '3.1.5'
```

Comment:
Usually, incrementing PATCH version is for bug fixes mostly. The correct would be incrementing the MINOR version: '3.2.0'.